### PR TITLE
More chrome string translation preparations

### PR DESCRIPTION
**Pull request by Mailaender**
These are also required for https://github.com/OpenRA/OpenRA/pull/20594.

### CODE REVIEW

**Comment by PunkPun** [comment]
Last commit should be merged into the first, or another way around where the yaml changes are moved to last commit

File: mods/cnc/chrome/settings-display.yaml [hunk 1]
```diff
@@ -288,7 +288,6 @@ Container@DISPLAY_PANEL:
 									Width: PARENT_RIGHT
 									Height: 20
 									Font: Regular
-									Text: Enable Frame Limiter
```
Comment (PunkPun) [inline mods/cnc/chrome/settings-display.yaml:291]: It's placed in the wrong commit

**Review by PunkPun — COMMENTED** [review]
Otherwise LGTM

**Review by PunkPun — DISMISSED** [review]

**Comment by PunkPun** [comment]
Needs a rebase

**Review by PunkPun — COMMENTED** [review]
As `LaberWithHighlightWidgets` is used by some buttons, you also need to go through `TooltipDesc:` and correct them. For example I see errors in `ingame.yaml`

**Comment by Mailaender** [comment]
Fixed and added the `<` brackets `>` to forbidden player names.

**Review by PunkPun — APPROVED** [review]

**Comment by PunkPun** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/67c1f925027c583fb8b6cabb74f2e11f5a1aab5f...07c8a0e78dc112aea30563f85d4b1b1ca7ef1662)

**Pull request merged**